### PR TITLE
e2e/snapshot:Fix spurious failure in snapshot test

### PR DESCRIPTION
**Pull request by rafikc30**
Change generic loopback device name to gluster specific

Signed-off-by: Mohammed Rafi KC <rkavunga@redhat.com>

### CODE REVIEW

File: e2e/lvmtest/lvmtest.go [hunk 1]
```diff
@@ -53,32 +53,32 @@ func createLV(num int, thinpoolSize, virtualSize string) error {
 	for i := 1; i <= num; i++ {
 		prefix := fmt.Sprintf("%s%d/%s", brickPrefix, i, lvmPrefix)
 		brickPath := fmt.Sprintf("%s_mnt", prefix)
-		devicePath := fmt.Sprintf("%s_loop", prefix)
+		devicePath := "/dev/gluster_loop" + strconv.Itoa(i)
```
Comment (Madhu-1) [inline e2e/lvmtest/lvmtest.go]: `/dev/gluster_loop` can this be defined as variable above?

File: e2e/lvmtest/lvmtest.go [hunk 2]
```diff
@@ -116,11 +116,15 @@ func deleteVHD(num int, force bool) error {
 	for i := 1; i <= num; i++ {
 		prefix := fmt.Sprintf("%s%d/%s", brickPrefix, i, lvmPrefix)
 		vhdPath := fmt.Sprintf("%s_vhd", prefix)
-		devicePath := fmt.Sprintf("%s_loop", prefix)
-		_, err := exec.Command("losetup", "-d", devicePath).Output()
-		if err != nil && !force {
+		devicePath := "/dev/gluster_loop" + strconv.Itoa(i)
```
Comment (Madhu-1) [inline e2e/lvmtest/lvmtest.go]: same variable can be used here

File: e2e/lvmtest/lvmtest.go [hunk 3]
```diff
@@ -138,16 +142,16 @@ func createVHD(num int, size string) error {
 	for i := 1; i <= num; i++ {
 		prefix := fmt.Sprintf("%s%d/%s", brickPrefix, i, lvmPrefix)
 		vhdPath := fmt.Sprintf("%s_vhd", prefix)
-		devicePath := fmt.Sprintf("%s_loop", prefix)
+		devicePath := "/dev/gluster_loop" + strconv.Itoa(i)
```
Comment (Madhu-1) [inline e2e/lvmtest/lvmtest.go]: same here

File: e2e/lvmtest/lvmtest.go (hunk 1 above)
Comment (kshlm) [reply e2e/lvmtest/lvmtest.go]: A constant at the top of the package would be better.

Comment (kshlm) [reply e2e/lvmtest/lvmtest.go]: A constant at the top of the package would be better.

**Pull request merged**